### ISSUE TITLE
Collection manager can't see private works

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#3411]
### Descriptive summary
Version: Hyrax 3.0

When a user views the admin page for a collection that they manage, they can't see private works that have been deposited and added to the collection by another user.

### Expected behavior

Collection manager should be able to view all items deposited into a collection that they manage, regardless of visibility settings.

### Actual behavior

Collection manager can't see private works.  The number of works in the Works section on the collection admin page does not reflect the private works in the collection.

### Steps to reproduce the behavior

1. Create a private collection and assign two accounts to collection manager and depositor roles
2. Depositor account deposits a work with private visibility to the collection
3. Manager account navigates to the collection admin page and views the Works section.  The private work is not visible and the number of works does not reflect the private work.

**Comment by chrisdaaz** [comment]
i just confirmed this on `nurax-dev`. i added a manager and depositor participant to a collection, deposited a private work to a collection and logged in as the manager of that collection and could not see the work that was deposited (i could see that the collection contains two works that are not visible to the manager, however):

![image](https://user-images.githubusercontent.com/24395592/48565752-e64c7200-e8be-11e8-838b-5b1a000f7e29.png)

**Comment by rjkati** [comment]
The "total items" count does display the two hidden works.  However, the "Works" label at the bottom of the page does not display the correct number of works in the collection.

![works](https://user-images.githubusercontent.com/32644254/48567901-35e56a80-e8cc-11e8-9067-f6a455b4af5c.PNG)

**Comment by no-reply** [comment]
I suspect this is by-design, and that this is very much *not* the expected behavior.

> Collection manager should be able to view all items deposited into a collection that they manage, regardless of visibility settings.

User collections are a common use case, and shouldn't override item-level permissions, right?

cc: @vantuyls, @elrayle, @samvera/hyrax-repo-managers

**Comment by vantuyls** [comment]
I think the expectations are these:
1) Collection manager can see all works in the collection, regardless of visibility of said works
2) User can only see works in the collection that User has rights to see, based on the visibility of the works

Related: do we think that the collection manager for a parent collection should or should not be able to see all works in a child collection?

**Comment by julesies** [comment]
oh i remember this now... @rjkati how did you add works to the collection?  It doesn't work as expected and I do think this is confusing but i believe:

--IF you add a work, click the relationship tab, select collection, then the permissions are not conferred.

--However, IF you go to the collection dashboard and add the work from THAT direction, permissions are conferred.

**Comment by julesies** [comment]
it also depends on the collection type

http://samvera.github.io/collection-type-participants.html#configuring-collection-sharing

and 

http://samvera.github.io/collection-sharing.html#understanding-collection-sharing

**Comment by rjkati** [comment]
@julesies I added the work via the relationship tab into a User Collection.  It's very confusing.

**Comment by no-reply** [comment]
@rjkati I can't see in your screenshot which collection is under discussion here. Can you tell us which Collection and which Works are behaving differently than you expect?

Looking at @chrisdaaz's `New collection for rc3_qa`, this is a "User Collection" which has both the permissions sharing feature and the visibility feature turned off. My understanding is that membership in this collection should never impact item permissions.

**Comment by no-reply** [comment]
> Ahhh I didn't understand that. And how would one know, or what would they do if they made a mistake and wanted to change which collection it was in and had permissions transferred in?

good questions. I think the present answer is: they don't know, and there is no way they could retroactively assign the correct permissions.

Maybe someone else has a happier diagnosis?

**Comment by rjkati** [comment]
@no-reply 

Collection: https://nurax-dev.curationexperts.com/collections/ff365540b?locale=en

Works: https://nurax-dev.curationexperts.com/concern/generic_works/fq977t840?locale=en
https://nurax-dev.curationexperts.com/concern/generic_works/n583xv21t?locale=en

**Comment by no-reply** [comment]
https://nurax-dev.curationexperts.com/collections/ff365540b also looks like a "User Collection". My understanding is that this collection type explicitly has permission inheritance turned off.

**Comment by chrisdaaz** [comment]
that's correct. it's because of the `User Collection` settings

![image](https://user-images.githubusercontent.com/24395592/49119388-c5e0c800-f26d-11e8-804e-05cbc1b6f099.png)

by default, the `APPLY TO NEW WORKS` option is checked when creating a new collection type. 

- I created a new collection type: `User Collection for Sharing Private Works with Managers`
- I created a [collection using that collection type](https://nurax-dev.curationexperts.com/dashboard/collections/qb98mf57c?locale=en).
- I added a manager to that collection (still as a repo admin).
- I added two private works to that collection _that already existed in the repository_
- I checked for them as the manager: they were **not** accessible
- I added a new private work to that collection _that did not already exist in the repository_
- I checked for it as the manager: it was accessible

it's confusing, but i **think** it's working as expected..
